modules/nf-core/samtools/calmd/tests/main.nf.test.snap

```diff
@@ -22,25 +22,25 @@
     "bam_stub_versions": {
         "content": [
             [
-                "versions.yml:md5,6cdfd8646cf5fb016bd57422bfa00243"
+                "versions.yml:md5,51ce5935fb227e0d66d293442d3a2efd"
             ]
         ],
         "meta": {
             "nf-test": "0.8.4",
-            "nextflow": "24.01.0"
+            "nextflow": "23.10.1"
         },
-        "timestamp": "2024-02-13T16:12:24.279177284"
+        "timestamp": "2024-05-28T15:42:47.344537089"
     },
     "bam_versions": {
         "content": [
             [
-                "versions.yml:md5,6cdfd8646cf5fb016bd57422bfa00243"
+                "versions.yml:md5,51ce5935fb227e0d66d293442d3a2efd"
             ]
         ],
         "meta": {
             "nf-test": "0.8.4",
-            "nextflow": "24.01.0"
+            "nextflow": "23.10.1"
```

Comment:
@matthdsm : I used nf 23.10.1 when updating the test.snaps. Would it have been better to use 24.01.0 as that was what was previously used for these tests?

Reply:
I think it should match the nf version that's used in to CI?

Reply:
Oh, seems like the CI tests are using the latest NF, that is, v24.4.2 which was released two hours ago :-) I'll try to see if I can update the test.snaps with NF v24.4.2.